src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -577,12 +577,14 @@ public static JsonElement CreateNextLink(string path, NameValueCollection? query
         /// </summary>
         /// <param name="jsonResult">Results plus one extra record if more exist.</param>
         /// <param name="first">Client provided limit if one exists, otherwise 0.</param>
+        /// <param name="Default">Default limit if none provided.</param>
```

Comment:
Update documented params to match actual

Reply:
fixed.